src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -97,6 +95,33 @@ export const getReservesGuardData = formData => {
   };
 };
 
+/**
+* Converts the treatment date range into an array of objects from just an object
+* @param {object} treatmentDateRange object containing from/to date range
+* @returns {array} of treatmentDateRange's
+*/
+const transformDateRange = (treatmentDateRange) => {
+  return [treatmentDateRange];
+};
+
+/** Note: This function will be moved to the 526EZ helper file
```

Comment:
Since this is in the 526EZ helper file, we can delete this.

Reply:
Fixed!